AWSAppSyncClient/Internal/AWSMutationRetryNotifier.swift

```diff
@@ -54,7 +58,7 @@ final class AWSMutationRetryHelper: Cancellable {
     
     private func scheduleTimer(at deadline: DispatchTime) {
         nextSyncTimer = DispatchSource.makeOneOffDispatchSourceTimer(deadline: deadline, queue: handlerQueue) {
-            AppSyncLog.debug("Timer fired, attempting mutation operation")
+            AppSyncLog.debug("Timer fired, attempting mutation operation. Retry number: \(self.retryAttemptNumber)")
```

Comment:
nit, store `retryAttemptNumber` outside closure to avoid referencing `self` within it.